### ISSUE TITLE
working2

### ISSUE THREAD

**Issue by zb360732** [talhaanwarch/ODIR2019#1]
Hello, I would like to know if running working2 alone is sufficient to achieve the same final result as yours. I tried running working2, but my valacc is only 35, which is quite different from your 58.

**Comment by talhaanwarch** [comment]
try this @zb360732 
https://github.com/talhaanwarch/course/blob/main/multi%20label.ipynb

**Comment by hukaiwlw** [comment]
Hello, I have run the code using epoches of 20, 
because the original test has no labels, we split the 3500 annotation datasets as 2240/560/700 for train/valid/test.
train_data = df[df['Dataset'] == 'train']
train_remaining, test_indices = train_test_split(
    train_data.index, 
    test_size=700, 
    random_state=42
)

train_indices, val_indices = train_test_split(
    train_remaining, 
    test_size=0.2, 
    random_state=42
)
I found the best result of 20 epoches is 
Kappa: 0.2557 | AUC: 0.7447 | F1: 0.3417 | Final Score: 0.4473
Is this right?

**Comment by hukaiwlw** [comment]
"kappa": 0.5484336363210259,
"AUC_vlaue": 0.9167666361765955,
"f-1_score": 0.90125,
"Final_Score": 0.7888167574992071

I'm confused for this results reported in the link [https://github.com/talhaanwarch/ODIR2019?tab=readme-ov-file](url) ,is this result from 700test from 3500annotation datasets, or reported from the competition? Why I cannot get such a result using working2.ipynb solution.

**Comment by talhaanwarch** [comment]
I could not recall accurately, but the score i mentioned is from test set from the odir challenge. i think i also did some ensembling
